### ISSUE TITLE
Issues with .eval() for batchnorm

### ISSUE THREAD

**Issue by ehannigan** [fxia22/pointnet.pytorch#12]
I ran across a few problems when running show_seg.py with the newest version of pytorch. 
Two problems in show_seg.py
- batchnorm needs a batchsize>1. I solved this problem by adding .eval() to show_seg.py
- There also seems to be a difference in the dim data.max() returns. 

One problem in pointnet.py
- PointNetDenseCls uses log_softmax() which now requires a dim parameter

I have fixed them, but I am new to using github and I've never done a pull request before. How could I submit the changes?
